sos/plugins/kubernetes.py

```diff
@@ -22,33 +22,75 @@ class kubernetes(Plugin, RedHatPlugin):
     """Kubernetes plugin
     """
 
-    option_list = [("podslog", "capture logs for pods", 'slow', False)]
+    packages = ('kubernetes',)
```

Comment:
*facepalm* / *sigh* :-)

What a terrible package name... OK, so we just have to live with this - I would add a short comment in the plugin file e.g. "NOTE: Red Hat Atomic $VERSION packages Kubernetes in the atomic-openshift package".

I know it probably seems very obvious at the moment but it may not to whoever is looking at it in 2-3 years time.